mapiproxy/util/oc_memcached.h

```diff
@@ -0,0 +1,51 @@
+/*
+   Memcached util functions
+
+   OpenChange Project
+
+   Copyright (C) Jesús García Sáez 2015
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef __OC_MEMCACHED_H__
+#define __OC_MEMCACHED_H__
+
+#include <libmemcached/memcached.h>
+
+#define MSTORE_MEMC_DFLT_HOST "127.0.0.1"
+#define MSTORE_MEMC_DFLT_PORT 11211
+
+/**
+ *  \details Returns a memcached_st pointer to use it in all libmemcached
+ *  functions. Internally this connections will be reused between different
+ *  calls (it will be one connection per config_string).
+ *
+ *  \param config_string Config string to initialize a memcached context,
+ *  in case of being NULL it will default to MSTORE_MEMC_DFLT_HOST. For more
+ *  info about format check online doc of libmemcached:
+ *
+ *      http://docs.libmemcached.org/libmemcached_configuration.html
+ */
+memcached_st *oc_memcached_new_connection(const char *config_string);
```

Comment:
doxygen content to be moved in source file rather than header to match global usage of doxygen